__tests__/app.test.js

```diff
@@ -26,5 +26,34 @@ describe('app', () => {
           });
         });
     });
+    test('404: GET should return error message when given route that does not exist', () => {
+      return request(app)
+        .get(`/api/notARoute`)
+        .expect(404)
+        .then(({body}) => {
+          expect(body.msg).toBe('Path Not Found');
+        });
+    });
+  });
+  describe('GET /api/reviews', () => {
+    test('200: GET should return an array of review objects', () => {
+      return request(app)
+        .get(`/api/reviews`)
+        .expect(200)
+        .then(({body}) => {
+          expect(body.reviews).toHaveLength(13);
```

Comment:
This is a good test for this endpoint 😄 You could also write something to check the instance is an array 👍